e2e-tests/constants/examples/postcode-examples.js

```diff
@@ -0,0 +1,14 @@
+export const INVALID_POSTCODES = {
+  ONE_LETTER_FIRST_PART: 'S 2AA',
+  NO_LETTERS_FIRST_PART: '22 2AA',
+  TWO_DIGITS_SECOND_PART: 'SW1 22A',
+  THREE_DIGITS_SECOND_PART: 'SW1 222',
+  ALL_LETTERS_SECOND_PART: 'SW1 AAA',
+  TOO_MANY_CHARACTERS: 'SW1A 2AAA',
+  TOO_FEW_CHARACTERS: 'S 2AA',
```

Comment:
For the too many & too few tests, should we have tests for with spaces and without spaces?